conf/simulator/nps/nps_sensors_params_default.h

```diff
@@ -30,12 +30,27 @@
 #define NPS_BODY_TO_IMU_THETA  IMU_BODY_TO_IMU_THETA
 #define NPS_BODY_TO_IMU_PSI    IMU_BODY_TO_IMU_PSI
 
+// try to determine propagate frequency
```

Comment:
Should this not depend on the IMU instead of the AHRS/INS? I think some INS/AHRS (ekf2?) may implement a callback for a gyro sensor update, but propagate the whole filter a slower rate.

Reply:
The exact IMU freq is not known, it depends of many sensor-dependent settings. So using the propagation frequency is the best guess I think. I made it configurable, so it can be changed with `NPS_XXXX_DT` if needed.